src/sys/socket/addr.rs

```diff
@@ -721,6 +767,15 @@ impl SockAddr {
             SockAddr::Netlink(..) => AddressFamily::Netlink,
             #[cfg(any(target_os = "ios", target_os = "macos"))]
             SockAddr::SysControl(..) => AddressFamily::System,
+            #[cfg(any(target_os = "android", target_os = "linux"))]
+            SockAddr::Ether(..) => AddressFamily::Packet,
+            #[cfg(any(target_os = "dragonfly",
+              target_os = "freebsd",
```

Comment:
These aren't aligned properly, the `target_os` part should all be vertically aligned.